client/src/FormFrontend/FormFrontend.yaml

```diff
@@ -6,6 +6,13 @@ form_frontend_privacy_ack:
   en: I accept
   fr: J'accepte
 
+form_frontend_faq_note:
+  en: Please read and acknowledge the FAQ before submitting.
+  fr: S'il vous plaît lisez et reconnaissez les FAQ avant de soumettre le formulaire.
+form_frontend_faq_ack:
+  en: I acknowledge that I read the FAQ
+  fr: Je reconnais que j'ai lu les FAQ
```

Comment:
Minor comment here, I checked on some other GC sites how they used the FAQ term in french and from what I can tell it is most commonly referred to as la "foire aux questions" without the use of the acronym, might be something to consider? As a french reader, just using the acronym feels a bit awkward and out of place. I would suggest using the full name in french and putting the acronym in parentheses afterwards. @Stephen-ONeil can probably comment further. If we do end up just using the FAQ acronym, it would at least have to be changed to "la FAQ".

Reply:
There is also the other acronym QSP (questions souvent posées) that is sometimes used

Reply:
I'll defer to you! If you could, please check the other instance of FAQ in french text, maybe open a PR with your suggested edits and let Christina know what convention you settle on. My only thought is to not bother with the acronym in parenthesis if it's always spelled out in full for french.